infrastructure/arm_templates/gw-nsg/parameters.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Note that I'm preserving the ARM templates from the export for now. These are used to generate the Bicep files, though I am not committing the decompiled bicep files that I am using as inspiration for mine.

I don't think it is particularly worth reading through the ARM files Jump to `infrastructure/resource_group_level/main.bicep.`